docs/bokeh/source/_templates/sections/footer.html

```diff
@@ -35,7 +35,7 @@ <h6>Documentation</h6>
         <h6>Hands On</h6>
         <ul class="nav flex-column">
           <li class="nav-item">
-            <a class="nav-link" href="https://mybinder.org/v2/gh/bokeh/bokeh-notebooks/master?filepath=tutorial%2F00%20-%20Introduction%20and%20Setup.ipynb">Tutorials</a>
+            <a class="nav-link" href="https://github.com/bokeh/tutorial/">Tutorials</a>
```

Comment:
@bryevdv I changed to link in the footer, too. Is this correct? The link seems to work (sometimes).

Reply:
Yes I think we should update/change to remove all binder links. It does sometimes work but also e.ge is sometimes down for days at a time (last week)